lib/end.go

```diff
@@ -42,6 +44,15 @@ func (a *ACBuild) End() error {
 
 	err = os.RemoveAll(a.ContextPath)
 	if err != nil {
+		switch err1 := err.(type) {
+		case *os.PathError:
+			switch err2 := err1.Err.(type) {
+			case syscall.Errno:
+				if err2 == syscall.EACCES {
+					fmt.Fprintf(os.Stderr, "you'll need to run this as a user with appropriate privileges\n")
```

Comment:
Same as above. Also, please turn the message into an impersonal form removing the "you", see https://github.com/coreos/docs/blob/master/STYLE.md#eschew-you